ctapipe/containers.py

```diff
@@ -700,8 +700,8 @@ class ParticleClassificationContainer(Container):
         nan,
         (
             "prediction of the classifier, defined between "
-            "[0,1], where values close to 0 are more "
-            "gamma-like, and values close to 1 more "
+            "[0,1], where values close to 1 are more "
```

Comment:
If I am not missing something, this is now inverted to how the models is getting trained

Reply:
You missed something :D